docs/catalog/cloud/use-serverless.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:small
+ - serverless
+---
+
+# Use serverless cloud services
+
+## Description
+
+Serverless cloud services are services that the cloud provider manages for the application. These scale dynamically with the workload needed to fulfill the service task and apply best practices to keep resource usage minimal.
+
+## Solution
+
+Adapt serverless services for the application.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation. using serverless cloud services will impact:
+
+- `E`: Serverless architectures share resources and only use resources when needed, this reduces the embodied carbon.
```

Comment:
This should impact
M -  as servers are launched based on the request, and provisioning is taken care of by the cloud provider.

Not clear on Intensity. Pls clarify - 
Unless we integrate with a carbon intensity provider and plan to run workloads based it based on time.